services/src/main/java/org/apache/unomi/services/impl/segments/SegmentServiceImpl.java

```diff
@@ -163,7 +158,6 @@ public void postConstruct() throws IOException {
         }
         bundleContext.addBundleListener(this);
         initializeTimer();
-        loadScripts();
```

Comment:
Is it related to the current PR?

Reply:
Yes I moved the painless script loading in the persistence service :)